index.src.html

```diff
@@ -752,10 +757,7 @@ <h4 id="algorithm-require-mediation" algorithm>Require User Mediation</h4>
      Exposed=Window,
      SecureContext]
     interface PasswordCredential : Credential {
-      attribute USVString idName;
-      attribute USVString passwordName;
-
-      attribute CredentialBodyType? additionalData;
+      attribute USVString password;
```

Comment:
I think we should make this `readonly`, just like the `FederatedCredential` attributes.